pyvrp/cpp/search/LocalSearch.h

```diff
@@ -16,6 +16,31 @@ namespace pyvrp::search
 {
 class LocalSearch
 {
+public:
+    /**
+     * Simple data structure that tracks statistics about the number of local
+     * search moves applied to the most recently improved solution.
+     *
+     * Attributes
+     * ----------
+     * num_moves
+     *     Number of evaluated node and route pair moves.
+     * num_improving
+     *     Number of applied (improving) moves.
+     */
+    struct Statistics
```

Comment:
What kind of additional statistics should/might we want to track? Number of evaluated and applied moves seems to me to be the biggest one, but there might be more I haven't thought of yet.